utils/i2c-tools/Makefile

```diff
@@ -44,6 +45,7 @@ define Package/i2c-tools
   CATEGORY:=Utilities
   TITLE+=tools for Linux
   DEPENDS:=+libi2c
+  VARIANT:=bin
```

Comment:
I don't think this needs to be a VARIANT.

Reply:
but I'm also not sure how this behaves if you remove the `VARIANT` assignment

Reply:
I think that this is good approach comparing it to not having variant because without it the build directory would be `-i2c-tools-$(PKG_VERSION)` (notice initial dash). Or build dir would have to be set conditionally to variant being defined. I don't like that and I am afraid of some other possibly unexpected behavior.

Reply:
ack
thanks for clarifying